scripts/common.sh

```diff
@@ -0,0 +1,26 @@
+#/!bin/bash
+# Various functions used by other scripts
+
+message() {
+  echo -e "\e[35m\e[1m**************************    $*    ********************************\e[0m"
+}
+
+tezos_client() {
```

Comment:
Changed from `tezos-client` to `tezos_client` so that we can `source` this file.